### PR TITLE
Please merge THEMVE-3054

### PR DESCRIPTION
**Pull request by exosite-winnie**

### CODE REVIEW

File: createScript/createStableUser.py [hunk 1]
```diff
@@ -0,0 +1,177 @@
...
+            key = role.keys()
+            key.remove('name')
+            userType = key[0]
+
+            variables = {
+                "role":{
+                    "group_id": self.groupId,
+                    "name": role['name'],
+                    "permissions": []
+                    }
```
Comment (exosite-RonHsieh) [inline createScript/createStableUser.py]: Too much

File: createScript/createStableUser.py [hunk 2]
```diff
@@ -0,0 +1,177 @@
...
+                }
+            }
+        '''
+        groups = self.execute(query)
+        for group in groups['data']['groups']:
+            if group['name'] == 'root':
+                self.groupId = group['id']
+        if self.groupId == '':
+            print ("Get group root failed")
+            return
```
Comment (exosite-RonHsieh) [inline createScript/createStableUser.py]: What is this use?

File: createScript/createStableUser.py (hunk 1 above)
Comment (exosite-winnie) [reply createScript/createStableUser.py]: I can't understand...
What too much?

File: createScript/createStableUser.py [hunk 3]
```diff
@@ -0,0 +1,177 @@
...
+        
+        for user in userList:
+            variables = {
+                "invite":{
+                    "email": user,
+                    "group_id": self.groupId,
+                    "role_id": ''
+                }
+            }
+
```
[the remark this answers is not in the document]
Comment (exosite-winnie) [reply createScript/createStableUser.py]: I don't understand

File: createScript/createStableUser.py [hunk 4]
```diff
@@ -0,0 +1,158 @@
...
+                    self.viewerRole = roleId['data']['createRole']['id']
+                elif userType == 'tester':
+                    self.testerRole = roleId['data']['createRole']['id']
+                else:
+                    self.managerRole = roleId['data']['createRole']['id']
+            else:
+                print ('Create role failed')
+
+    def create_user(self):
+        userList = {'testing+manager@exosite.com', 'testing+tester@exosite.com','testing+viewer@exosite.com'}
```
Comment (exosite-RonHsieh) [inline createScript/createStableUser.py]: please space in front of 'testing+viewer@exosite.com'

**Pull request merged**